src/cmodel.c

```diff
@@ -1380,6 +1380,8 @@ cmodel_t *CM_LoadMap (char *name, qbool clientload, unsigned *checksum, unsigned
 
 	COM_FileBase (name, loadname);
 
+	Hunk_TempFlush();
```

Comment:
not obvious, an explanation is required.

Reply:
As CalcChecksum has run through the lumps and used up some of the temp area, I wanted to reset to baseline before performing the actual load to maximize heap available as after here there are multiple lumps in memory at the same time, vs the checksum where it only needs to deal with one at a time.

Reply:
I understand why you did it, but it took time without a comment. A brief comment would not hurt why it is required.
Like: 
`// Force flush hunk after CM_CalcChecksum since CM_ReadLump use Hunk_TempAllocMore, that provides maximum hunk memory before heavy loading.`